### PR TITLE
Add Monoflex 60 keyboard

### PR DESCRIPTION
**Pull request by key10iq**
## Description
Add Monoflex 60 to QMK. It is a QMK and VIA-compatible 60% keyboard.
Monoflex 60 GB was hosted by SantiGo on [GH](https://geekhack.org/index.php?topic=110738.0) over a year ago, but it failed because of the runner (SantiGo). Now people are trying to save this GB (they want to buy almost finished cases from the factory, which the runner didn't pay until the end)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Added new keyboard - monoflex60 (QMK and VIA)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/monoflex60/monoflex60.h [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ * │10   │11 │12 │13 │14 │15 │16 │17 │18 │19 │1a │1b │1c │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐ 1d │
+ * │20    │21 │22 │23 │24 │25 │26 │27 │28 │29 │2a │2b │2c │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │30  │31 │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3c        │
+ * ├────┴┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤
+ * │40   │41 │43   │47                         │4b   │4c │4d   │
+ * └─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘
+ */
+#define LAYOUT_iso( \
```
Comment (drashna) [inline keyboards/monoflex60/monoflex60.h:90]: This looks like it matched `LAYOUT_60_iso` from the community layouts.  If so, then community layout support should be added (eg, the name changed, and `LAYOUTS = 60_iso` added to the rules). 

Additionally, the LAYOUT_tsangan looks to match the `LAYOUT_60_tsangan_hhkb` community layout, as well.

Comment (key10iq) [reply keyboards/monoflex60/monoflex60.h:90]: It matched `LAYOUT_60_iso_tsangan`. I'll fix it.

**Review by drashna — APPROVED** [review]

File: keyboards/monoflex60/readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* Hardware Supported: Monoflex 60 PCB
+* Hardware Availability: [GB](https://geekhack.org/index.php?topic=110738.0)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make monoflex60:default
+	
+Flashing example for this keyboard:
+
+	make monoflex60:default:flash
```
Comment (fauxpark) [inline keyboards/monoflex60/readme.md:19]:
```suggestion
    make monoflex60:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by key10iq** [comment]
@drashna sorry for the ping too, but......

**Pull request merged**